.prettierrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    // all default
```

Comment:
at now it's works only for typescript, by my understanding it have to work only for js. Okay, so I suggest to extend prettier config with `overrides` section, all other settings leave as default. As sample feel free to relate on https://github.com/abahachuk/ptushki-backend/pull/3, the same regarding `.prettierignore`.

I'm haven't yet experience with `ts` with `eslint`, but believe that are some adjustments needed in `.eslintrc` file to make prettier work with `js`, sure that it comes about `plugins`. 

Even more: it's issue not with Prettier, but with ESLint, try to create error specific to `eslint` in `service-worker.js`. But this also creates issue because some of this `js` files comes as configs, and even have not to be compiled. Again, it more about how eslint simultaneously support as `js` and as `ts` files: or there needed other plugins, or multiply parsers. Now I only confirm that it not work together.

Reply:
regarding `overrides` section, please, note that prettier supports `css` and `scss`, so it will looks like:
```json
    {
      "files": "*.scss",
      "options": {
        "parser": "scss",
        "singleQuote": true,
        "tabWidth": 4
      }
    },
```

probably then we should `stylelint` it pairs with prettier a well))

Reply:
Thanks for thorough review!

Regarding prettier not working by itself: 
I purposefully didn't set up prettier to work standalone - only as an extension to eslint. 
It works well for ts{x} files when running as a plugin for eslint here, and you probably noticed there is no separate prettier tasks in package.json scripts.

My reasoning:
We already have "lint"and "lint-fix" tasks for our linting needs. Allowing to use both eslint and prettier would introduce multiple ways to do one thing, and bring inconsistencies to devs workflows.
So i decided to have only eslint tasks, and use prettier as a eslint plugin only.

Reply:
regarding service-worker.js. It comes with cra setup, and i decided not to bother reformatting it

Reply:
> Allowing to use both eslint and prettier would introduce multiple ways to do one thing, and bring inconsistencies to devs workflows.

I didnt speak about using them separately, f.e. `tslint`, `eslint` or even `stylelint` have plugins & configs to work together and call `prettier` under hood. But there other files like `.json`, '.md', `.eslintrc` (as `json`) & etc, that havent linters, but have to be prettified and prettier support them.

For this you can extend config for those type of files and then you will be able by IDE prettify them. Next step is to add `lint-staged` to not worry about IDE or manually call eslint, or prettier directly. Just take a look at https://github.com/abahachuk/ptushki-backend/pull/3, if havent chance still, how fine all works together.